### DISCUSSION TITLE
Some information don't output correctly

### DISCUSSION THREAD

**Discussion by 5ouma** [fastfetch-cli/fastfetch#501] [Q&A] [answered]
It doesn't output some information compared to [neofetch](https://github.com/dylanaraps/neofetch).
- WM: I use `Rectangle`, but it says `Quartz Compositor`.
- Terminal: I use `iTerm2`, but it says nothing.
- Terminal Font: I use `FiraCode-SemiBold 18` and `HackGen35ConsoleNF-Bold 18`, but it says nothing.

All of the above are output by neofetch.
Could you tell me how to solve this?

**Comment by CarterLi** [comment, marked as answer]
> WM: I use Rectangle, but it says Quartz Compositor

It's not detected by default. Specifying `--allow-slow-operations` should work

> Terminal: I use iTerm2, but it says nothing.

It works for me, but something else might go wrong.

Please report it to issues and follow the bug report instructions. I can't debug it without more detailed information.

> Terminal Font: I use FiraCode-SemiBold 18 and HackGen35ConsoleNF-Bold 18, but it says nothing.

Terminal Font detection requires a successful Terminal detection.

**Reply by 5ouma** [reply]
Thank you for your reply!

I think the problem Terminal doesn't show as I expected is because of using [Fig](https://fig.io).
It says `zsh (figterm)` and `Unknown terminal`.
How can I make it recognize not Fig but iTerm?

Also, I ran `fastfetch` without running `Fig`, it says correctly for Terminal as `iTerm 3.5.0beta10`.
But for Terminal Font, it just says for `FiraCode-SemiBold (18pt)`, my main font, and it doesn't say `HackGen35ConsoleNF-Bold (18pt)`, my sub font.

Neofetch says `Terminal Font: FiraCode-SemiBold 18 (normal) / HackGen35ConsoleNF-Bold 18 (non-ascii)`.

**Reply by CarterLi** [reply]
All these issues should be fixed in the latest dev build

**Reply by 5ouma** [reply]
Yes!
I confirmed it worked when I build from source on the dev branch.
Thank you for your commits!
I'm looking forward to the next release.

**Comment by CarterLi** [comment]
Note: [`Quartz Compositor`](https://en.wikipedia.org/wiki/Quartz_Compositor) is always the system level [compositing window manager](https://en.wikipedia.org/wiki/Compositing_window_manager) (with the process name `WindowServer`) on modern macOS system. That is to say, without Quartz Compositor, you can't even open a single window.

`Rectangle`, on the other hand, is a window arrangement tool. It can be called a window manager but it's not the same level of Quartz Compositor. Fastfetch chose to print `Quartz Compositor` because it matches the detection results of other systems (kwin, mutter, DWM, etc), while `Rectangle` and some other tools can be detected as a plugin of Quartz Compositor with `--allow-slow-operations`

**Reply by 5ouma** [reply]
Thank you. It works as expected for WM when I add the option.
